SConstruct

```diff
@@ -1338,6 +1338,23 @@ env.AlwaysBuild(
             env.PrettyCommand('TIDY', 'src', 'yellow')
         )))
 
+if 'PKG_CONFIG_PATH' in env.Dictionary():
```

Comment:
Let's replace direct `env` checks with `env.HasArg` calls, like:

```
if env.HasArg('PKG_CONFIG_PATH')
```

It'd allow the user to specify the variables in either of these forms:

```
VAR=foo scons
scons VAR=foo
```

We'll also need `env.OverrideFromArg` call here: https://github.com/roc-project/roc/blob/453631ccbd1cb9ac657877faef2997cee8a78d46/SConstruct#L313-L315

Like:

```
env.OverrideFromArg('PKG_CONFIG', default='pkg-config')
```